app/views/root/index.html.erb

```diff
@@ -0,0 +1 @@
+<div id="root">
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/207516678-9f66840d-a3e2-4822-82ae-c2aa1b64e8e7.png)

- [ ] Your API is working and giving random quote :blush: but your front end part is not showing anything. Kindly fix it :blush: Your react app is not working properly :thinking:

Reply:
Thank you @HAMMAS-SALEEM for reviewing my project, :+1: 
but on my side I am running ```rails db:seed``` and ```./bin/dev``` in my terminal and it's working on my side
![image](https://user-images.githubusercontent.com/102757126/207529273-d849d8eb-6257-48af-8a4a-5b64da56b16b.png)